package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@parifi/sdk",
-  "version": "1.0.12",
+  "version": "1.0.31-dev",
```

Comment:
Update the sdk version to use `2.0.0` to indicate major changes with the switch to Synthetix